src/seeds.proposals.cpp

```diff
@@ -89,16 +107,37 @@ uint64_t proposals::get_size(name id) {
   }
 }
 
+void proposals::prop_size_change(name id, int64_t delta) {
+  size_tables sizes(get_self(), get_self().value);
+  auto sitr = sizes.find(id.value);
+  if (sitr == sizes.end()) {
+    sizes.emplace(_self, [&](auto& item) {
+      item.id = id;
+      item.size = delta;
+    });
+  } else {
+    uint64_t newsize = sitr->size + delta; 
+    if (delta < 0) {
+      if (sitr->size < -delta) {
+        newsize = 0;
+      }
+    }
+    sizes.modify(sitr, _self, [&](auto& item) {
+      item.size = newsize;
+    });
+  }
+}
+
 void proposals::onperiod() {
     require_auth(_self);
 
     auto pitr = props.begin();
 
     auto prop_majority = config.get(name("propmajority").value, "The propmajority parameter has not been initialized yet.");
-    uint64_t quorum = config.find(name("propquorum").value)->value;
 
     uint64_t number_active_proposals = 0;
-    uint64_t total_eligible_voters = get_size("active.sz"_n);
+    uint64_t total_eligible_voters = get_size("active.sz"_n, contracts::accounts);
```

Comment:
active.sz should be in this contract's sizes as well - no need to leave than in the accounts sizes table?

Reply:
Yes, you are right, there is no need to user the account's sizes table. I left only the proposals' sizes table.
Just one doubt, does it means the accounts' sizes table should no be updated? I mean in the active.sz

Reply:
I changed it in master branch to that active size simply tracks voice table size. 

Only users that can vote are in the voice table. When someone gets demoted from citizen to resident, they are also removed from the voice table. So it's a lot simpler and more accurate to just keep track of voice. 

@IanMendozaJaimes do you agree? Or did I miss something?